Week7/CurrentNews/app/src/main/java/com/florencenjeri/cocktailsrecipe/network/ApiBuilder.kt

```diff
@@ -1,28 +1,29 @@
-package com.florencenjeri.cocktailsrecipe.networking
+package com.florencenjeri.cocktailsrecipe.network
 
 import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
 import kotlinx.serialization.json.Json
 import okhttp3.Interceptor
 import okhttp3.MediaType.Companion.toMediaType
 import okhttp3.OkHttpClient
-import okhttp3.Response
 import okhttp3.logging.HttpLoggingInterceptor
 import retrofit2.Retrofit
 
-val API_KEY = "n5YlnB5zNgFR84AHAMJg-oKSGgGShCohW24yIkJW2cu3fOrQ"
-
-val  BASE_URL = "https://api.currentsapi.services/"
+const val API_KEY = "n5YlnB5zNgFR84AHAMJg-oKSGgGShCohW24yIkJW2cu3fOrQ"
+private const val apiKey = "apiKey"
```

Comment:
I would change the name to full upper case, and rename this to "KEY_AUTHORIZATION" or "KEY_API".